src/components/__tests__/MenuItem.spec.tsx

```diff
@@ -35,26 +33,32 @@ describe('Components', () => {
         title: 'Menu item title'
       }
 
-      const component = shallow(<MenuItem {...menuItemProps} />)
-      expect(component.find('[testID="title"]').props().children).toBe(menuItemProps.title)
+      const { queryByText } = render(<MenuItem {...menuItemProps} />)
+      const title = queryByText('Menu item title')
+      expect(title).not.toBeNull()
     })
 
     it('should render children passed to it', () => {
       const menuItemProps: IMenuItemProps = {
         ...defaultMenuItemProps
       }
 
-      const component = shallow(<MenuItem {...menuItemProps} />)
-      expect(component.contains(menuItemProps.children)).toBe(true)
+      const { queryByText } = render(<MenuItem {...menuItemProps} />)
+      const title = queryByText('Text of children')
+      expect(title).not.toBeNull()
     })
 
     it('should render black arrow icon when selected is false', () => {
       const menuItemProps: IMenuItemProps = {
         ...defaultMenuItemProps
       }
 
-      const component = shallow(<MenuItem {...menuItemProps} />)
-      expect(component.find('[testID="arrow"]').props().fill).toBe(COLORS.lunesBlack)
+      const { getByTestId, getByText } = render(<MenuItem {...menuItemProps} />)
+      const arrowIcon = getByTestId('arrow')
+      expect(arrowIcon.props.fill).toBe(COLORS.lunesBlack)
+      const title = getByText('MenuItemTitle')
+      // @ts-expect-error
+      expect(title._fiber.pendingProps.style[0].color).toBe(COLORS.lunesGreyDark)
```

Comment:
Any better idea how to test for styles?

Reply:
See here: https://github.com/digitalfabrik/lunes-app/blob/7fe3012bcd9ee05ad5f428b0bd3b325a58e8e157/src/components/__tests__/MenuItem.spec.tsx
I accidentally refactored MenuItem already, as the tests were failing due to StyledComponents and I didn't know you were already working on it. As it was not part of my issue though I reverted and let you have the honor. Still, maybe the changes can help you here :)

Reply:
how about title.instance.props.style[0].color? :)